### PR TITLE
Course explorer updates

### PR DESCRIPTION
**Pull request by Erin007**
With the new site navigation redesign, we needed to update the Course Explorer on studio.code.org/courses. This PR incorporates description changes, consolidates CSF into one course rather than listing each sub course separately and moves the Accelerated Course bar.  "Learn more" links have been updated to teal with a chevron to be more consistent with other redesign pages. 

BEFORE: 
course explorer before

AFTER: 
course explorer - line break before prereqs

* There was off-line discussion about whether the Accelerated Course bar should be lighter and aligned with the Computer Science Fundamentals as it is in the [spec](https://docs.google.com/document/d/1W0ayYaTNEv_SIQyuWnqdgzSjTreaqRBsnOIrqFAwnEM/edit#heading=h.4u91umsv39ho). I've highlighted these changes on the doc, and will let @breville have final say on what this should look like.

### CODE REVIEW

**Comment by Erin007** [comment]
cc: @joshlory

File: shared/haml/course_explorer_table.haml [hunk 1]
```diff
@@ -19,13 +19,10 @@
...
+- courses << { name: "CS Fundamentals", starts: 0, ends: 5, description: "Introductory course for early readers.", group: "20 hour courses for each grade can be taught once a week", link: "/educate/curriculum/elementary-school", img: "/shared/images/courses/logo_tall_course1.jpg", color: "#33E0EF", long: "Code.org’s CS Fundamentals courses blend online, self-guided, and self-paced tutorials with “unplugged” activities that require no computer at all. Students will create computer programs that will help them learn to collaborate with others, develop problem-solving skills, and persist through difficult tasks. They will study programming concepts, computational thinking, digital citizenship, and develop interactive games or stories they can share. Students at every grade level (including pre-readers) can begin CS Fundamentals with no prior programming experience." }
```
Comment (breville) [inline shared/haml/course_explorer_table.haml]: it might be worth renaming `group` to `subheading` or something like that.

Comment (Erin007) [reply shared/haml/course_explorer_table.haml]: Yeah, that does make more sense.

**Comment by tanyaparker** [comment]
Can we put a line break in the long descriptions between the big paragraph and the "No prerequisites" line? Like if you look at my picture in the doc. Want to call that out a little more.

File: shared/haml/course_explorer_table.haml [hunk 2]
```diff
@@ -19,13 +19,10 @@
...
+- courses << { name: "CS Principles", starts: 9, ends: 12, description: "A year-long rigorous introductory computer science course at the high-school level covering the Internet, data, and programming. Can be run as a non-AP course.", subtitle: " Full year course", link: "/educate/csp", img: "/shared/images/teacher-announcement/csp-widgets.png", color: "#00adbc", long: "In fall 2016, the College Board launched its newest AP® course, AP Computer Science Principles. The course introduces students to the foundational concepts of computer science and challenges them to explore how computing and technology can impact the world. The AP Program designed AP Computer Science Principles with the goal of creating leaders in computer science fields and attracting and engaging those who are traditionally underrepresented with essential computing tools and multidisciplinary opportunities. The curriculum is flexible to be taught as a normal course or as an AP course. No prerequisites needed. This course works for beginners or students with experience in CS Fundamentals or CS Discoveries."}
```
Comment (tanyaparker) [inline shared/haml/course_explorer_table.haml]: Is description used for anything? Or is it just long? Would rename long to description and get rid of the other.

**Comment by tanyaparker** [comment]
I haven't checked all of the text, but it'll be easier for me to test once it's merged in. I will definitely confirm then.

**Comment by Erin007** [comment]
@tanyaparker  I added a line break before the prereqs and updated the PR screenshot. What do you think?

**Pull request merged**